compareECLFiles.cmake

```diff
@@ -263,20 +263,22 @@ add_test_compareECLFiles(CASENAME polymer_oilwater
                          FILENAME 2D_OILWATER_POLYMER
                          SIMULATOR flow
                          ABS_TOL ${abs_tol}
-                         REL_TOL ${rel_tol})
+                         REL_TOL ${rel_tol}
+                         TEST_ARGS --tolerance-mb=1.e-7 )
 
 add_test_compareECLFiles(CASENAME polymer_injectivity
                          FILENAME 2D_POLYMER_INJECTIVITY
                          SIMULATOR flow
                          ABS_TOL ${abs_tol}
-                         REL_TOL ${rel_tol})
+                         REL_TOL ${rel_tol}
+                         TEST_ARGS --tolerance-mb=1.e-7 --tolerance-wells=1.e-6)
```

Comment:
Please also test if you can remove the --tolerance-wells

Reply:
from the previous PR, no for this case, because it is the last one added. Anyway, I can create a PR to verify.

Reply:
Sorry, then I misunderstood your previous comment. Please try removing --tolerance-mb=1.e-7 then.

Reply:
It is not sufficient, which was tested already.

Reply:
anyway, I will create a test just in case.

Reply:
If I understand correctly, the test will break at your end if one of them is removed, regardless which?

Reply:
Yes. I believe so. And I could not verify for only `--tolerance-mb=1.e-7 ` for now , due to the following PR did not manage to update the reference results due to compilation error.  But with only `--tolerance-wells=1.e-6` it does not work. 

https://github.com/OPM/opm-simulators/pull/2404

Reply:
I see, yes, please check it out. If the material balance is sufficient there too, we have one common switch for all polymer cases. That is just about as simple as we can get it.

Reply:
For this case, it needs both `--tolerance-mb=1.e-7` and `--tolerance-wells=1.e-6` on my machine. 

As stated previously, adding these two arguments likely break some other people's local regression test.